packages/v4/src/content/design-guidelines/content/writing-design-documentation.md

```diff
@@ -33,7 +33,7 @@ Always write headings in sentence case. Write component names in lowercase.
 
 Example: 
 
-"Usage guidelines for card views"
+> "Usage guidelines for card views"
 
 ### Bolds and italics
```

Comment:
Wonder if this should just be called "Bolding" bc you don't mention italics in the body?

Reply:
I know this was super complicated in the end, so maybe just ignore me and don't indent.. sorry for the back and forth, just looks weirder with the block quote thing it's doing (and I know that's not intentional)

Reply:
Added a line of italics guidance, since we do use them occasionally!